transaction-flow.md

```diff
@@ -151,13 +147,22 @@ When a sender is created you will receive a response which contains the senders
 * rejected - The sender has failed sanction list checks (cannot transact)
 * disabled - A sender is put into this state as a result of a delete request via the API (cannot transact)
 
+In order to transact with Bitpesa we need to have an `approved` sender record. The flow for approving senders depend on whether KYC requirements are waived for your integration or not (See the notes section).
+
+In case the KYC requirements are not waived then the typical flow for approval this will be the following:
+
+![Sender registration flow](uml/sender-kyc.png)
+
+In case the KYC requirements are waived then all created senders will be in the `approved` state immediately, and can be immediately used for transactions.
+
 Notes:
 
 * The sender's phone number is composed of two parts, the `phone_country` (in ISO 2-letter format), and the `phone_number`. The phone number should be specified without the international prefix.
-* The `documents` should contain all documents necessary to KYC the sender. If you already do KYC on your system, then please contact us and we can waive this requirement from you. In this case you can send us an empty list of documents: `[]`. In case you don't do KYC on your site, then you will need to send us documents, for this please see the [API reference documentation](https://api.bitpesa.co/documentation#documents)
+* The `documents` should contain all documents necessary to KYC the sender. If you already do KYC on your system, then please contact us and we can waive this requirement from you. In this case you should send us an empty list of documents: `"documents": [ ]` in the request. In case you don't do KYC on your site, then you will need to send us documents, for more details on this please see the [API reference documentation](https://api.bitpesa.co/documentation#documents).
```

Comment:
These are still a bit muddled - maybe change it so that one bullet point refers only to KYC-waived, and one refers only to KYC required:

* The `documents` parameter should contain all documents necessary to KYC the sender.
  * If you already do KYC on your system, then please contact us and we can waive this requirement from you. In this case, you should send us an empty list of documents - `"documents": []` - in the request. All of the senders you create in the system will be immediately set to the `approved` state and you won't need to wait for them to get approved.
   * In case you don't do KYC on your site, then you will need to send us documents. For more details on this please see the [API reference documentation](https://api.bitpesa.co/documentation#documents).